data/src/main/java/com/egiwon/data/di/NetworkModule.kt

```diff
@@ -0,0 +1,51 @@
+package com.egiwon.data.di
+
+import com.egiwon.data.datasource.RandomImageService
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
+import okhttp3.Interceptor
+import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+import javax.inject.Singleton
+
+@Module
+@InstallIn(SingletonComponent::class)
+class NetworkModule {
+
+    @Provides
+    @Singleton
+    fun provideHttpLoggingInterceptor() = HttpLoggingInterceptor().apply {
+        level = HttpLoggingInterceptor.Level.BODY
+    }
+
+    @Provides
+    @Singleton
+    fun provideGsonConverterFactory(): GsonConverterFactory = GsonConverterFactory.create()
+
+    @Provides
+    @Singleton
+    fun provideHttpClient(
+        loggingInterceptor: HttpLoggingInterceptor,
+        interceptor: Interceptor
+    ) =
+        OkHttpClient.Builder()
+            .addInterceptor(loggingInterceptor)
+            .addInterceptor(interceptor)
+            .build()
```

Comment:
```suggestion
    ) = OkHttpClient.Builder()
            .addInterceptor(loggingInterceptor)
            .addInterceptor(interceptor)
            .build()
```
P3. 불필요한 라인은 제거해주면 어떨까요?